### ISSUE TITLE
False positive arity error linting a cljc file with a reader literal

### ISSUE THREAD

**Issue by mynomoto** [borkdude/clj-kondo#500]
**version**
clj-kondo v2019.09.23-alpha-SNAPSHOT

**platform**
linux

**problem**
False positive arity error linting a cljc file with a reader literal

**repro**
`src/cljkondo.cljc`
``` clojure
(ns cljkondo
  (:require
    [schema.core :as sc]))

(sc/check t/Uuid #uuid "00000000-0000-0000-0000-000000000000")
```
`deps.edn`
``` clojure
{:deps  {prismatic/schema {:mvn/version "1.1.11"}}
 :paths ["src"]}
```
**expected behavior**
No linter warnings

**actual behavior**
```
$ clj-kondo --lint "$(clj -Spath)"
src/cljkondo.cljc:5:1: error: schema.core/check is called with 1 arg but expects 2
linting took 2558ms, errors: 1, warnings: 0

**Comment by mynomoto** [comment]
Renaming the file to `.clj` or `.cljs` makes the error disapear.

**Comment by borkdude** [comment]
Good catch. Fixed with https://github.com/borkdude/clj-kondo/commit/89abdb1b80e9b1def538a5119fe31ff2bf209451. Just in time for the next release, I'll create it tomorrow morning.

**Comment by mynomoto** [comment]
> Just in time for the next release, I'll create it tomorrow morning.

This is what I was aiming for, thanks!!
